content/recipes/swagger.md

```diff
@@ -200,71 +300,33 @@ Now you can start your server with the following command:
 $ npm run start
 ```
 
-Navigate to `http://localhost:3000/api/cats` to see SwaggerUI for your cats:
+Navigate to `http://localhost:3000/api/cats` to see Swagger UI for your cats:
 
 <figure><img src="/assets/swagger-cats.png" /></figure>
 
-While `http://localhost:3000/api/dogs` will expose a SwaggerUI for your dogs:
+While `http://localhost:3000/api/dogs` will expose a Swagger UI for your dogs:
 
 <figure><img src="/assets/swagger-dogs.png" /></figure>
 
-> warning **Notice** You have to construct a **SwaggerOptions** with `DocumentBuilder`, run `createDocument()` against newly constructed `options` then immediately "serve" it with `setup()` before you can start working on a second **SwaggerOptions** for a second Swagger Specification. This specific order is to prevent Swagger configurations being overridden by different options.
-
-#### Working with enums
-
-To be able for `SwaggerModule` to identify an `Enum`, we have to manually set the `enum` property on `@ApiModelProperty` with an array of values.
-
-```typescript
-@ApiModelProperty({ enum: ['Admin', 'Moderator', 'User']})
-role: UserRole;
-```
-
-`UserRole` enum can be defined as following:
-
-```typescript
-export enum UserRole {
-  Admin = 'Admin',
-  Moderator = 'Moderator',
-  User = 'User',
-}
-```
-
-> warning **Note** The above usage can only be applied to a **property** as part of a **model definition.**
-
-Enums can be used by itself with the `@Query()` parameter decorator in combination with the `@ApiImplicitQuery()` decorator.
-
-```typescript
-@ApiImplicitQuery({ name: 'role', enum: ['Admin', 'Moderator', 'User'] })
-async filterByRole(@Query('role') role: UserRole = UserRole.User) {
-  // role returns: UserRole.Admin, UserRole.Moderator OR UserRole.User
-}
-```
-
-<figure><img src="/assets/enum_query.gif" /></figure>
-
-> warning **Hint** `enum` and `isArray` can also be used in combination in `@ApiImplicitQuery()`
-
-With `isArray` set to **true**, the `enum` can be selected as a **multi-select**:
-
-<figure><img src="/assets/enum_query_array.gif" /></figure>
-
-#### Working with arrays
+#### Tags
 
-We have to manually indicate a type when the property is actually an array:
+In order to attach a controller to a specific tag, use the `@ApiTags(...tags)` decorator.
 
 ```typescript
-@ApiModelProperty({ type: [String] })
-readonly names: string[];
+@ApiTags('cats')
+@Controller('cats')
+export class CatsController {}
 ```
 
-Simply put your type as the first element of an array (as shown above) or set an `isArray` property to `true`.
+#### Headers
 
-#### Tags
-
-At the beginning, we created a `cats` tag (by making use of `DocumentBuilder`). In order to attach the controller to the specified tag, we need to use `@ApiUseTags(...tags)` decorator.
+In order to define custom headers that are expected as part of the request, use `@ApiHeader()`.
```

Comment:
In order to define custom headers ->
To define custom headers <- remove **In order**